lib/daru/plotting/nyaplot/dataframe.rb

```diff
@@ -30,6 +32,15 @@ def plot opts={}, &block
 
         private
 
+        def x_axis_undefined?(opts)
+          !(opts[:x] || opts.keys.any? { |k| k.to_s.match(/x\d+/) })
```

Comment:
Just a minor nit-picking from my part. 😄 

This format ...
```ruby
x_axis_defined = cond_1 || cond_2
index_as_default_x_axis(opts) unless x_axis_defined
```
... seems a bit more readable, when compared to the current format ...
```ruby
x_axis_undefined = !(cond_1 || cond_2)
index_as_default_x_axis(opts) if x_axis_undefined
```

Reply:
Ok, done.